src/analysis/analytic/basic.lean

```diff
@@ -312,28 +312,66 @@ lemma has_fpower_series_at.coeff_zero (hf : has_fpower_series_at f pf x) (v : fi
 let ⟨rf, hrf⟩ := hf in hrf.coeff_zero v
 
 /-- If a function admits a power series expansion, then it is exponentially close to the partial
-sums of this power series on strict subdisks of the disk of convergence. -/
-lemma has_fpower_series_on_ball.uniform_geometric_approx {r' : ℝ≥0}
+sums of this power series on strict subdisks of the disk of convergence.
+
+This version provides an upper estimate that decreases both in `∥y∥` and `n`. See also
+`has_fpower_series_on_ball.uniform_geometric_approx` for a weaker version. -/
+lemma has_fpower_series_on_ball.uniform_geometric_approx' {r' : ℝ≥0}
   (hf : has_fpower_series_on_ball f p x r) (h : (r' : ennreal) < r) :
   ∃ (a ∈ Ioo (0 : ℝ) 1) (C > 0), (∀ y ∈ metric.ball (0 : E) r', ∀ n,
-  ∥f (x + y) - p.partial_sum n y∥ ≤ C * a ^ n) :=
+    ∥f (x + y) - p.partial_sum n y∥ ≤ C * (a * (∥y∥ / r')) ^ n) :=
 begin
-  obtain ⟨a, ha, C, hC, hp⟩ : ∃ (a ∈ Ioo (0 : ℝ) 1) (C > 0), ∀ n, ∥p n∥ * r' ^n ≤ C * a^n :=
```

Comment:
I prefer the previous version, as it says what `hp` is. Why did you change it?

Reply:
Reverted.